clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_pipeline.py

```diff
@@ -304,7 +304,12 @@ def build_core_nodes(self):
             initrand=self.parameters["initrand"],
         )
         # Susceptibility distortion correction using T1w image
-        sdc = epi_pipeline(name="SusceptibilityDistortionCorrection")
+        sdc = epi_pipeline(
+            self.base_dir,
+            self.parameters["delete_cache"],
```

Comment:
Right now this line is breaking the instantiation tests (`KeyError` for `delete_cache`). I think we should add a default setting in `check_pipeline_parameters` (line 47), and maybe set `delete_cache` in the test.